lib/tether/tether.go

```diff
@@ -252,6 +257,41 @@ func (t *tether) setMounts() error {
 			return fmt.Errorf("unsupported volume mount type for %s: %s", k, v.Source.Scheme)
 		}
 	}
+	return t.populateVolumes()
```

Comment:
Ah I see, hence the Dir call we talked about above. makes sense, sorry about that.